homeassistant/components/yeelight/light.py

```diff
@@ -380,6 +386,17 @@ def _power_property(self):
     def _predefined_effects(self):
         return YEELIGHT_MONO_EFFECT_LIST
 
+    @property
+    def state_attributes(self):
```

Comment:
You're not allowed to override `state_attributes`. Implement `device_state_attributes` instead.

Reply:
Thanks, pushed fix.